plugins/linux-alsa/alsa-input.c

```diff
@@ -27,6 +27,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include <alsa/pcm.h>
 
 #include <pthread.h>
+#include <string.h>
```

Comment:
why this include which you don't use ? same observation for all the files

Reply:
If I had to guess, debug print text that has been removed, but forgetting to remove the corresponding header include.

Reply:
Strictly speaking <string.h> is where memset() is declared. Although this is one of those things that is almost always picked up in some upstream #include in other headers, since I was not testing on all platforms I thought it safe to put it here just to be sure. I've been bitten before on sillier things.

Reply:
Would you mind instead using `= {0}`, this is supported on all compilers we compile on and doesnt require any headers and is currently more popular than memset in the codebase.